### ISSUE TITLE
Allow absolute path to device in `zenfs` utility

### ISSUE THREAD

**Issue by metaspace** [westerndigitalcorporation/zenfs#246]
Paths to devices in ZenFS have to be given without leading `/dev/` prefix. This is a source of confusion for new users. It is also not documented.

We should allow `zenfs` utility and RocksDB URI to use absolute paths for devices.

**Comment by damien-lemoal** [comment]
Yes ! That will also allow using /dev/disk/by-id/xxx as these are stable device names that do not depend on the scan order on boot.

**Comment by metaspace** [comment]
We might also have issues with using `/dev/mapper/...` with the current implementation.
